kong/dao/schemas/upstreams.lua

```diff
@@ -61,19 +175,26 @@ return {
       type = "number",
       default = DEFAULT_SLOTS,
     },
+    healthchecks = {
+      type = "table",
+      schema = gen_schema(healthchecks_defaults),
+      default = healthchecks_defaults,
```

Comment:
from a UX perspective, what happens when I want to define a `healthchecks` sub-schema with mostly default data, but a few changes? do i need to manually fill out the whole `healthchecks_defaults` as part of my request?

Reply:
The above `gen_schema()` function generates fields that have the `default` attribute set to the one defined in the above stub `healthchecks_defaults`, so you should indeed be able to partially specify health check configuration and expect default values to be inserted when necessary.